src/main/java/com/basho/riak/client/api/commands/kv/MultiCommand.java

```diff
@@ -371,9 +371,9 @@ public void await() throws InterruptedException
         }
 
         @Override
```

Comment:
Up on line 331 above, there's another call to await with a timeout.  This is the only place I can find left in the code (there are a few in the tests too) that calls await with a timeout but ignores the return value.  Because it checks isDone afterwards, ignoring the return value does seem OK, though perhaps a comment would help.

For what it's worth, the test code ignores the return value here:

https://github.com/basho/riak-java-client/blob/expand-coverage-api/src/test/java/com/basho/riak/client/api/commands/itest/ITestMultiDelete.java#L40

https://github.com/basho/riak-java-client/blob/expand-coverage-api/src/test/java/com/basho/riak/client/core/operations/itest/ITestListBucketsOperation.java#L159

https://github.com/basho/riak-java-client/blob/expand-coverage-api/src/test/java/com/basho/riak/client/core/operations/itest/ITestListKeysOperation.java#L177